apps/files/docs/extensionpoints.adoc

```diff
@@ -17,3 +17,74 @@
 
 OC.$extend.request('files', 'file-details-panel', payload).then( response => {});
 ----
+
+### Files list status indicators
+
+An extension can extend status indicators in files list with custom component, properties and methods.
+Extending this list can be done by creating a new object inside of appInfo called `filesListIndicators`. This object can contain keys `name` and `component`.
+
+#### Props
+
+Please see known issues before using props in custom indicators.
+
+- `item`: resource to which are indicators assigned
+- `parentPath`: parent folder path of the item
+
+#### Example:
+
+##### App entry point
+[source,js]
+----
+const appInfo = {
+  name: 'ExampleIndicators',
+  id: 'ExampleIndicators',
+  icon: 'info',
+  isFileEditor: false,
+  extensions: [],
+  filesListIndicators: [{
+    name: 'CustomIndicators',
+    component: Indicators
+  }]
+}
+----
+
+##### Indicators component
+[source,vue]
+----
+<template>
+  <oc-button
+    v-if="isFolder"
+    class="file-row-share-indicator uk-text-middle"
+    aria-label="Custom indicator"
+    @click="triggerAlert"
+    variation="raw"
+  >
+    <oc-icon
+      name="folder"
+      class="uk-text-middle"
+      size="small"
+      variation="active"
+    />
+  </oc-button>
+</template>
+
+<script>
+  export default {
+    name: 'Indicators',
```

Comment:
isn't that the "ExampleIndicators" component ?

Reply:
Ah, you meant renaming the component? It still is called just Indicators. The name inside of appinfo is an arbitrary string and not tied to the name of the component.

Reply:
in that case please do rename it to something else to avoid the confusion with the same component(s) of Phoenix

my requests here are mostly about removing any possible confusion for someone reading the code the first time

Reply:
Renamed